neurokit2/signal/signal_fixpeaks.py

```diff
@@ -563,67 +624,92 @@ def _plot_artifacts_lipponen2019(artifacts, info):
 # NeuroKit
 # =============================================================================
 def _remove_small(
-    peaks, sampling_rate=1000, interval_min=None, relative_interval_min=None, robust=False
+    peaks,
+    sampling_rate=1000,
+    interval_min=None,
+    relative_interval_min=None,
+    robust=False,
 ):
     if interval_min is None and relative_interval_min is None:
         return peaks
-
     if interval_min is not None:
-        interval = signal_period(peaks, sampling_rate=sampling_rate, desired_length=None)
+        interval = signal_period(
+            peaks, sampling_rate=sampling_rate, desired_length=None
+        )
         peaks = peaks[interval > interval_min]
-
     if relative_interval_min is not None:
-        interval = signal_period(peaks, sampling_rate=sampling_rate, desired_length=None)
+        interval = signal_period(
+            peaks, sampling_rate=sampling_rate, desired_length=None
+        )
         peaks = peaks[standardize(interval, robust=robust) > relative_interval_min]
-
     return peaks
 
 
 def _interpolate_big(
-    peaks, sampling_rate=1000, interval_max=None, relative_interval_max=None, robust=False
+    peaks,
+    sampling_rate=1000,
+    interval_max=None,
+    relative_interval_max=None,
+    robust=False,
 ):
     if interval_max is None and relative_interval_max is None:
         return peaks
-
-    continue_loop = True
-    while continue_loop is True:
-        if interval_max is not None:
-            interval = signal_period(peaks, sampling_rate=sampling_rate, desired_length=None)
-            peaks, continue_loop = _interpolate_missing(
-                peaks, interval, interval_max, sampling_rate
-            )
-
-        if relative_interval_max is not None:
-            interval = signal_period(peaks, sampling_rate=sampling_rate, desired_length=None)
-            interval = standardize(interval, robust=robust)
-            peaks, continue_loop = _interpolate_missing(
-                peaks, interval, relative_interval_max, sampling_rate
-            )
-
+    if interval_max is not None:
+        interval = signal_period(
+            peaks, sampling_rate=sampling_rate, desired_length=None
+        )
+        peaks, continue_loop = _interpolate_missing(
```

Comment:
With the while loop gone, `_interpolate_missing()` doesn't have to return `continue_loop` anymore.

Reply:
Thanks for catching that!